src/js/main.js

```diff
@@ -18,24 +18,35 @@ let app;
 let loadedConfig;
 
 // Retrieve the config by API key.
-let appName = getAppName();
-if (!appName) {
+let appName = getParam('appName');
+let jwKey = getParam('jwKey');
```

Comment:
I would recommend renaming the parameter to `key` in order to be consistent with the Showcase specification.